charts/newrelic-k8s-metrics-adapter/values.yaml

```diff
@@ -43,17 +51,25 @@ config:
   #   removeClusterFilter: false
 
 image:
-  registry:
+  # -- The container registry.
+  registry: docker.io
```

Comment:
I think leaving the registry blank would be nice for users which use custom CRI configs @kang-makes

Reply:
Good, I removed it